packages/line/src/hooks.js

```diff
@@ -18,8 +18,8 @@ export const useLineGenerator = ({ curve }) => {
         () =>
             line()
                 .defined(d => d.x !== null && d.y !== null)
-                .x(d => d.x)
-                .y(d => d.y)
+                .x(d => d.x | 0)
```

Comment:
Is this intentional? It seems to take only the integer part of the value: 

![image](https://github.com/BitBloomTech/nivo/assets/1688189/4f71d85c-e442-4cf6-9aa7-3037ef412d1d)

Perhaps `||` was intended?

![image](https://github.com/BitBloomTech/nivo/assets/1688189/27bb49fb-1044-4415-9f29-c9004ebbbadf)

Reply:
It is intentional, truncate to integer. Not using floating point pixels gave a measurable improvement.